action.yaml

```diff
@@ -68,8 +73,17 @@ runs:
     - id: record_test
       shell: bash
       run: |
-        launchable record tests \
-          --build ${{ inputs.build_name }} \
-          --allow-test-before-build \
-          ${{ inputs.test_runner }} \
-          ${{ inputs.report_path }}
+        if [ "${{ inputs.no_build }}" == "false" ]; then
```

Comment:
nits: Wouldn't it be easier to understand it the other way around?

```
if [ "${{ inputs.no_build }}" == "true" ]; then
  launchable record tests --no-build
else 
  launchable record tests 
fi
```

Reply:
fixed
https://github.com/launchableinc/record-build-and-test-results-action/pull/5/commits/7b233720d29a3c764e9cfa9d040e7befe338bf4a